aspnetcore/data/ef-rp/read-related-data.md

```diff
@@ -31,57 +32,87 @@ The following illustrations show the completed pages for this tutorial:
 
 There are several ways that EF can load related data into the navigation properties of an entity:
 
-* Eager loading. When the entity is read, its related data is retrieved. This typically results in a single join query that retrieves all of the data that's needed. Eager loading is specified in EF with the `Include` and `ThenInclude` methods.
+* [Eager loading](https://docs.microsoft.com/ef/core/querying/related-data#eager-loading). Eager loading is when a query for one type of entity also loads related entities. When the entity is read, its related data is retrieved. This typically results in a single join query that retrieves all of the data that's needed. Eager loading is specified with the `Include` and `ThenInclude` methods.
```

Comment:
EF Core will issue multiple queries for some types of eager loading. This is actually a lot more efficient than was the case for some queries in EF6 where there was a single query.